src/Sql/Sql/help/Set-AzSqlServerAuditing.md

```diff
@@ -164,7 +165,7 @@ The recommended set of action groups to use is the following combination - this
 "SUCCESSFUL_DATABASE_AUTHENTICATION_GROUP",  
 "FAILED_DATABASE_AUTHENTICATION_GROUP"  
 This above combination is also the set that is configured by default. These groups cover all SQL statements and stored procedures executed against the database, and should not be used in combination with other groups as this will result in duplicate audit logs.
-For more information, see https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups.
+For more information, see https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups.  If you choose to override the default with your own configuration, you must first disable auditing on all destinations and then reapply each with the new configuration.
```

Comment:
I did not understand "reapply each with the new configuration", what do you mean by that?

Reply:
It was our experiences that the new configuration did not apply to all 3 targets, but each retained its own configuration.  Again, I will retest to confirm.

Just noting that I submitted the documentation change recommendation based on experience.  Below is the example script used; note, without first disabling, the AuditActionGroup remained set to the default:

$rg = "SR119040922002714"
$server = "sr119040922002714-sql"

$aag = "APPLICATION_ROLE_CHANGE_PASSWORD_GROUP","BACKUP_RESTORE_GROUP","DATABASE_LOGOUT_GROUP","DATABASE_OBJECT_CHANGE_GROUP","DATABASE_OBJECT_OWNERSHIP_CHANGE_GROUP","DATABASE_OBJECT_PERMISSION_CHANGE_GROUP","DATABASE_OPERATION_GROUP","DATABASE_PERMISSION_CHANGE_GROUP","DATABASE_PRINCIPAL_CHANGE_GROUP","DATABASE_PRINCIPAL_IMPERSONATION_GROUP","DATABASE_ROLE_MEMBER_CHANGE_GROUP","FAILED_DATABASE_AUTHENTICATION_GROUP","SCHEMA_OBJECT_ACCESS_GROUP","SCHEMA_OBJECT_CHANGE_GROUP","SCHEMA_OBJECT_OWNERSHIP_CHANGE_GROUP","SCHEMA_OBJECT_PERMISSION_CHANGE_GROUP","SUCCESSFUL_DATABASE_AUTHENTICATION_GROUP","USER_CHANGE_PASSWORD_GROUP";

Get-AzSqlServerAuditing -ResourceGroupName $rg -ServerName $server -LogAnalytics
#Set-AzSqlServerAuditing -State Disabled -ResourceGroupName $rg -ServerName $server -AuditActionGroup $aag -LogAnalytics -WorkspaceResourceId "/subscriptions/31555f3e-97ac-482f-937b-d07f9ce24599/resourcegroups/sr119040922002714/providers/microsoft.operationalinsights/workspaces/sr119040922002714-log"
Set-AzSqlServerAuditing -State Enabled -ResourceGroupName $rg -ServerName $server -AuditActionGroup $aag -LogAnalytics -WorkspaceResourceId "/subscriptions/31555f3e-97ac-482f-937b-d07f9ce24599/resourcegroups/sr119040922002714/providers/microsoft.operationalinsights/workspaces/sr119040922002714-log"
Get-AzSqlServerAuditing -ResourceGroupName $rg -ServerName $server -LogAnalytics

Reply:
@scott-epperly You are right, in order to modify `AuditActionGroup`, user needs to disable and then enable audit policy. This is a bug, which will be fixed.
No need to update documentation, client should be able to update `AuditActionGroup` without the `disable` step.
Thanks for catching this!

Reply:
Thanks for the update!  Clearly, you can dismiss/cancel these requested documentation changes